deps/chakrashim/lib/chakra_shim.js

```diff
@@ -89,12 +120,34 @@
   // Parse 'stack' string into StackTrace frames. Skip top 'skipDepth' frames,
   // and optionally skip top to 'startName' function frames.
   function parseStack(stack, skipDepth, startName) {
+    var curr = parseStack;
+
     var splittedStack = stack.split('\n');
```

Comment:
`split` can take a Regex. How about sth. like `stack.split(/\([^\n]*\n/)` here? Maybe can make the code a bit simpler.